en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -72,16 +75,21 @@ The figure below shows how you can access your remote receiver (please find the
 ## Spektrum Satellite Receivers
 
 The V5 nano has a dedicated DSM cable.
-If using a Spektrum satellite receiver, this should be connected to the flight controller DSM/SBUS/RSSI interface. 
+If using a Spektrum satellite receiver, this should be connected to the flight controller `DSM/SBUS/RSSI` interface.
 
 
 ## Power
 
-The v5 nano kit includes HV\_PM, which supports 2~10S LiPo battery.
-Please connect the 6pin connector of the HW\_PM module to the flight control `Power` interface.
+The *v5 nano* kit includes the *HV\_PM* module, which supports 2~10S LiPo batteries.
+Connect the 6pin connector of the *HW\_PM* module to the flight control `Power` interface.
+
+> **Warning** The supplied power module is unfused (see [this issue](../flight_controller/cuav_v5_nano.md#issue_pm_unfused)).
 
 ![quickstart](../../assets/flight_controller/cuav_v5_nano/connection/v5_nano_quickstart_06.png)
 
+> **Note** The power module does not supply the output rail.
```

Comment:
@cuhome As discussed, added a note about powering the output rail. Please check.

Reply:
What is the output rail you are talking about?

Reply:
> the output rail.

Refers to the row of pins on the PWM header marked as +

Reply:
@cuhome FYI, to hopefully make this clear I changed the text to

> **Note** The power module is not a power source for peripherals connected to the PWM outputs.
  If you're connecting servos/actuators you will need to separately power them using a BEC.

It would still be great to show this rail being powered with an image.

Reply:
There is no problem with this description